src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp

```diff
@@ -684,6 +684,7 @@ namespace
         ICustEdit*              m_text_renderingthreads;
         ISpinnerControl*        m_spinner_renderingthreads;
         AppleseedRenderer*      m_renderer;
+        ICustEdit*              m_text_render_stamp;
```

Comment:
Keep data members ordered logically: `m_renderer` probably needs to stay at the end and `m_text_render_stamp` should stay close to the other settings.